File: kotlin/14.kt
fun main(){
    val userChoice : Int = readln().toIntOrNull() ?: 0;
    if (userChoice < 6 || userChoice > 10){
        println("Your number is less than 6, or greater than 10");
        return;
    } else {
        println("Your number is NOT less than 6, or greater than 10");}
}

File: dart/14.dart
import 'dart:io';

void main (){
  String userChoice = stdin.readLineSync()!;
  int userChoiceInt = int.parse(userChoice);
  if (userChoiceInt < 6 || userChoiceInt > 10){
    print("Your number is less than 6, or greater than 10");
  } else {
    print("Your number is between 6 and 10");
  }
}